lib/adapters/tool/testplane/index.ts

```diff
@@ -119,6 +126,13 @@ export class TestplaneToolAdapter implements ToolAdapter {
         return runner.run((collection) => this._tool.run(collection, {grep, sets, browsers, devtools, replMode}));
     }
 
+    async runWithoutRetries(...args: RunTestArgs): Promise<boolean> {
```

Comment:
I see that this method was just moved from the other file, but when do we need it, what's the use-case?

Reply:
It was moved from `lib/gui/app`, which is a common file for all adapters. But for other tools the change in the number of retries can be performed in completely different ways. For example for playwright I should specify option `--retries 0` in cli command, like this `playwright test --retries 0`.